lagom-cloud-object-storage-example/account-impl/src/main/java/com/lightbend/lagom/account/impl/readside/AccountExtractProcessor.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.account.impl.readside;
+
+import akka.Done;
+import akka.japi.Pair;
+import akka.stream.javadsl.Flow;
+import com.lightbend.lagom.account.impl.AccountEvent;
+import com.lightbend.lagom.javadsl.persistence.AggregateEventTag;
+import com.lightbend.lagom.javadsl.persistence.Offset;
+import com.lightbend.lagom.javadsl.persistence.ReadSideProcessor;
+import org.pcollections.PSequence;
+import org.pcollections.TreePVector;
+
+import javax.inject.Inject;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+public class AccountExtractProcessor extends ReadSideProcessor<AccountEvent> {
+
+  final private AccountExtractRepositoryImpl repository;
+
+  @Inject
+  public AccountExtractProcessor(AccountExtractRepositoryImpl repository) {
+    this.repository = repository;
+  }
+
+
+  @Override
+  public PSequence<AggregateEventTag<AccountEvent>> aggregateTags() {
+    return TreePVector.singleton(AccountEvent.TAG);
+  }
+
+
+  @Override
+  public ReadSideHandler<AccountEvent> buildHandler() {
+
+    return new ReadSideHandler<AccountEvent>() {
+
+
+      @Override
+      public CompletionStage<Done> globalPrepare() {
+        // we don't save offset, repo is designed to consume from first event each time
```

Comment:
If you implement a processor from scratch it has. You need to create the table where you read-model will be stored and the offset table. But indeed the comment makes no sense. It would be better to simply say: "No persistence. In-memory processor only"